website/docs/how-to/how-to-set-up-group-sso-sync.md

```diff
@@ -0,0 +1,41 @@
+---
+title: How set up user group SSO syncing
+---
+
+:::info availability
+
+User group syncing is planned to be released in Unleash 4.18 and will be available for enterprise customers.
```

Comment:
Same here as with the previous availability note. We may want to say that it's at "some point in the future".